pkg/asset/templates/kubelet.yaml

```diff
@@ -27,17 +27,11 @@ spec:
         - --api-servers={{ index .APIServers 0 }}
         - --config=/etc/kubernetes/manifests
         - --allow-privileged
-        - --hostname-override=$(MY_POD_IP)
```

Comment:
No problem with dropping the hostname override since hosts to run the kubelet-wrapper with an `/etc/resolv.conf` mount, so its not always using 8.8.8.8.